aspnetcore/mvc/controllers/areas.md

```diff
@@ -114,6 +114,10 @@ For more information, see [Routing to controller actions](xref:mvc/controllers/r
 
 To share a common layout for the entire app, move the *_ViewStart.cshtml* to the application root folder.
 
+### Shared tag helpers using the _ViewImports.cshtml file
+
+To share a common configuration of [tag helpers](xref:mvc/views/tag-helpers/intro) (eg: for `<a asp-action="Index">` to be converted to correct links), move the *_ViewImports.cshtml* to the application root folder.
```

Comment:
```suggestion
To ensure [tag helpers](xref:mvc/views/tag-helpers/intro) apply in your area ensure a proper *_ViewImports.cshtml* file exists at the application root or in your areas folder.
```

Reply:
Why not suggest to move the file that the template generates (as we do for layout)?

I don't have any data to back it up, but it seems more common to share *_ViewImports.cshtml* than to have separate for each area in an application.

Reply:
> Why not suggest to move the file that the template generates (as we do for layout)?

Ehh, _ViewImports can be used for many different reasons and for projects that have lots of TagHelpers developers typically want to separate out where certain TagHelpers are applicable.  In simple applications it's very common to only have one _ViewImports shared globally however in larger applications you'll see more _ViewImports interspersed throughout.

Reply:
I'm probably just more focused on helping others not spend hours to figure out why `<a asp-action="List">List</a>` doesn't work by default in an area, and suggest a quick fix. If you want to use ViewImports more heavily, you probably know their hirarctic nature and can adjust accordingly.

How about

 ### *_ViewImports.cshtml*
In its standard location, */Views/_ViewImports.cshtml* does not apply to areas. To use eg [tag helpers](xref:mvc/views/tag-helpers/intro), `@using` or `@inject` in your area, ensure a proper *_ViewImports.cshtml* file [apply to your area views](xref:/mvc/views/layout#importing-shared-directives). If you want the same behaviour in all your views, just move */Views/_ViewImports.cshtml* to application root.

Reply:
That sounds fine. I'd just make `apply` plural

Reply:
Ok. I changed `eg tag helpers, ...` to `common tag helpers, ....` as well.